2013/07/26 19:15:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii/page".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/page')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/portfolio/index.php?r=gii/page&view=about
---
2013/07/26 19:26:44 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "C:\xampp\htdocs\portfolio\assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in C:\xampp\htdocs\yii\framework\web\CAssetManager.php:138
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CAssetManager.php(123): CAssetManager->setBasePath('C:\xampp\htdocs...')
#1 C:\xampp\htdocs\yii\framework\web\CAssetManager.php(217): CAssetManager->getBasePath()
#2 C:\xampp\htdocs\yii\framework\gii\GiiModule.php(142): CAssetManager->publish('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\base\CComponent.php(111): GiiModule->getAssetsUrl()
#4 C:\xampp\htdocs\yii\framework\base\CModule.php(105): CComponent->__get('assetsUrl')
#5 C:\xampp\htdocs\yii\framework\gii\views\layouts\main.php(5): CModule->__get('assetsUrl')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div class="con...')
#10 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div class="con...')
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#12 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#13 C:\xampp\htdocs\yii\framework\gii\views\layouts\column1.php(7): CBaseController->endContent()
#14 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#15 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#16 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#17 C:\xampp\htdocs\yii\framework\gii\controllers\DefaultController.php(47): CController->render('login', Array)
#18 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionLogin()
#19 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#20 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#21 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#23 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/default/log...')
#24 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#25 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/portfolio/index.php?r=gii/default/login
---
2013/07/26 19:33:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'portfolio'' in C:\xampp\htdocs\yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 C:\xampp\htdocs\yii\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/portfolio/index.php?r=gii/model
HTTP_REFERER=http://localhost/portfolio/index.php?r=gii/default/index
---
2013/07/26 19:34:09 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'portfolio'' in C:\xampp\htdocs\yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 C:\xampp\htdocs\yii\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/portfolio/index.php?r=gii/model
---
2013/07/26 19:58:41 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('imagenesBlog/ad...')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portfolio/index.php?r=imagenesBlog/admin
HTTP_REFERER=http://localhost/portfolio/index.php?r=imagenesBlog/create
---
2013/07/28 19:35:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii/index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/index')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/portfolio/index.php?r=gii/index
---
2013/07/28 19:35:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "imagenesBlog/index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('imagenesBlog/in...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/portfolio/index.php?r=imagenesBlog/index
---
2013/07/28 19:35:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii/index".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/index')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/portfolio/index.php?r=gii/index
---
2013/07/28 19:51:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('login')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/login')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/portfolio/index.php?r=blog/login
---
2013/07/28 19:51:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "edir".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('edir')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('edir')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/edir')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/portfolio/index.php?r=blog/edir
---
2013/07/28 19:51:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "edit.php".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('edit.php')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('edit.php')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/edit.php')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/portfolio/index.php?r=blog/edit.php
---
2013/07/28 22:23:08 [error] [exception.CException] exception 'CException' with message 'Property "CActiveForm.enctype" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWidgetFactory.php(161): CComponent->__set('enctype', 'multipart/form-...')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(BlogController), 'CActiveForm', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(189): CBaseController->createWidget('CActiveForm', Array)
#3 C:\xampp\htdocs\portfolio\protected\views\blog\_form.php(13): CBaseController->beginWidget('CActiveForm', Array)
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\portfolio\protected\views\blog\create.php(18): CController->renderPartial('_form', Array)
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#12 C:\xampp\htdocs\portfolio\protected\controllers\BlogController.php(79): CController->render('create', Array)
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): BlogController->actionCreate()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#19 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#21 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#23 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('blog/create')
#24 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#25 C:\xampp\htdocs\portfolio\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/portfolio/index.php?r=blog/create
---
